app/templates/views/activity/notifications.html

```diff
@@ -12,17 +12,19 @@
       caption="Recent activity",
       caption_visible=False,
       empty_message='No messages found &thinsp;(messages are kept for {} days)'.format(limit_days)|safe,
-      field_headings=['Recipient', 'Status'],
+      field_headings=['Recipient', 'Status', 'Carrier', 'Response'],
```

Comment:
As we noted in Slack, we just want to adjust the header for the response status to be clear that it's coming from the carrier, e.g., like this:

```suggestion
      field_headings=['Recipient', 'Status', 'Carrier', 'Carrier Response'],
```

But feel free to adjust as you see fit!

Reply:
Updated the header in the notifications.html